<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoftTop</class>
 <widget class="QMainWindow" name="SoftTop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SoftTop - User Assistant</string>
  </property>
  <property name="windowIcon">
   <iconset theme="TT">
    <normaloff>iconTT.ico</normaloff>iconTT.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="topTable">
        <property name="cursor" stdset="0">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QGroupBox" name="sortByrGrBox">
          <property name="title">
           <string>Sort by</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="byHour">
             <property name="statusTip">
              <string>Show programs by last hour(date)</string>
             </property>
             <property name="text">
              <string>Hour</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="byDay">
             <property name="statusTip">
              <string>Show programs by last days</string>
             </property>
             <property name="text">
              <string>Day</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="byAllTime">
             <property name="statusTip">
              <string>Show most used program</string>
             </property>
             <property name="text">
              <string>All Time</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="updateButton">
             <property name="statusTip">
              <string>Update program list</string>
             </property>
             <property name="text">
              <string>Update Top</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="viewAllcheckBox">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>View all</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QDateEdit" name="dateEdit">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>648</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Program">
    <property name="title">
     <string>&amp;Program</string>
    </property>
    <addaction name="action_Close"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>&amp;Info</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionUpdate"/>
    <addaction name="separator"/>
    <addaction name="actionHour"/>
    <addaction name="actionDay"/>
    <addaction name="actionAll_Time"/>
   </widget>
   <addaction name="menu_Program"/>
   <addaction name="menuOptions"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>&amp;Update</string>
   </property>
   <property name="statusTip">
    <string>Update list</string>
   </property>
  </action>
  <action name="actionHour">
   <property name="text">
    <string>&amp;Hour</string>
   </property>
   <property name="statusTip">
    <string>Show programs by last hour(date)</string>
   </property>
  </action>
  <action name="actionDay">
   <property name="text">
    <string>&amp;Day</string>
   </property>
   <property name="statusTip">
    <string>Show programs by last days</string>
   </property>
  </action>
  <action name="actionAll_Time">
   <property name="text">
    <string>&amp;All Time</string>
   </property>
   <property name="statusTip">
    <string>Show most used program</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
